Pagina maquetado

gonziesc opened the pull request:
## Summary
- Principal page design: responsive and centered 
- Features: responsive text
-Footer: as design

## Screenshots
![selection_001](https://cloud.githubusercontent.com/assets/5241126/24708562/9c897628-19ed-11e7-93e6-f94e0d130c03.png)
![selection_002](https://cloud.githubusercontent.com/assets/5241126/24708563/9c90fdd0-19ed-11e7-8d58-e0e25758d1aa.png)
![selection_003](https://cloud.githubusercontent.com/assets/5241126/24708564/9c913a2a-19ed-11e7-87c8-edd36191e04e.png)

## Trello card
https://trello.com/c/zUGiAMmY/1-maquetar-la-seccion-pricipal-de-la-pagina

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+
```
- use 2 space indentation and indent elements properly when they are inside another element
- delete all blank lines between the tags
- classes should be in spinal-case instead of camelCase
- always use english for classes and Id's
- don't use `br` tags, thats a style thing and should be handled through css by setting a max-width to the paragraph
-  in the contact section elements should be wrapped inside a form

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,151 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,700');
+
+*{
```
- use 2 space indentation
- there should be a space before `{`
- there should be a space after the `:`
- check things are correctly indented
- there should be a single blank line between selectors and nowhere else.
- media queries should be added at the end of the file so they override the other styles placed before it if they have the same specificity
-

gonziesc commented:
## Summary
- indentation fixed
- blank spaces deleted
- classes changed to in spinal-case
- Id's in english
- replaced br tags with max-witdth property
- contact session wrapped in form 
- media added at the end of the css file 

## Screenshots
![screenshot from 2017-04-12 11-23-21](https://cloud.githubusercontent.com/assets/5241126/24962517/846ca5a8-1f72-11e7-8840-0146a979994f.png)
![screenshot from 2017-04-12 11-23-31](https://cloud.githubusercontent.com/assets/5241126/24962538/90a722e4-1f72-11e7-9a8f-d21068ecc0f7.png)


## Trello card
https://trello.com/c/zUGiAMmY/1-maquetar-la-seccion-pricipal-de-la-pagina

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,48 @@
+      <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+      <div class="border"></div>
```
this border should be added from css using the `border` property in the `.images` div I would use something like "main-section" or "main-container" instead of "images" to make it more understandable

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,48 @@
+      <div class="border"></div>
+      <div class="images">
+        <img class="phone" src="resources/phone.png" alt="phone">
+        <div class="border-line">
```
this looks a bit strange, you don't need those hr elements, you can set a width and height in pixels and set the background using a gradient color

SKOLZ replied:
```
background: linear-gradient(white, #f5c85f, white) ;

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,48 @@
+          <hr class="vertical-line">
+          <hr class="vertical-line-d">
+        </div>
+        <img  id="imagen" src="resources/tcg_logo.png" alt="logo">
```
id should be in english

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,48 @@
+        </div>
+        <img  id="imagen" src="resources/tcg_logo.png" alt="logo">
+      </div>
+      <div class="triangle">
```
this is purely aesthetic so it should be done through css. use a `::before` pseudo-element in the features to achieve this.

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,48 @@
+          <p class="description">
+            Get reminded of who owes you money or who you owe money to.
+          </p>
+          </div>
```
this is not correctly indented

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,48 @@
+        </div>
+        <form class="suscribe">
+          <h1 class="info">Get info on new updates</h1>
+          <input class="input-suscribe" type="text"></input>
```
since this is an email it should be `type='email'`

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,48 @@
+          <button class="button-suscribe">Subscribe</button>
+        </form>
+      </body>
+      </html>
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+  margin: 0;
+}
+
+.phone {
```
if it's empty delete it

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.border-line {
+  display: flex;
+	flex-direction: column;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.images {
+  display: flex;
+	justify-content: center;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.features {
+  display: flex;
+	justify-content: space-around;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.triangle {
+  height: 10px;
+	width: auto;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.texts {
+  display: flex;
+	flex-direction: column;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.suscribe {
+  display: flex;
+	flex-direction: column;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+	flex-direction: column;
+	justify-content: space-around;
+	align-items: center;
+	background-color: #EEEEEE;
```
colors that look like #AABBCC can be written this way: #ABC. in this case #EEE

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.title {
+  color: #111111;
+	text-align: center;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+}
+
+.description {
+	color: #999999;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+}
+
+.info {
+	font: Open Sans;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+}
+
+.button-suscribe {
+	border-radius: 8px;
```
this is not correctly indented

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+
+.button-suscribe {
+	border-radius: 8px;
+	border: 0.2px solid white;
```
use integer pixel values. 1px here

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,138 @@
+}
+
+.input-suscribe {
+	border-radius: 25px;
```
this is not correctly indented

gonziesc commented:
## Summary
- Border changed from div to css border
- linear-gradient changed to one line
- ::before pseudo-element used for triangles background
- Iinput type changed to email
- indentation fixes 


## Screenshots
![screenshot from 2017-04-18 11-33-23](https://cloud.githubusercontent.com/assets/5241126/25136276/ef7748b4-242a-11e7-8c90-58d654eb1560.png)
![screenshot from 2017-04-18 11-33-36](https://cloud.githubusercontent.com/assets/5241126/25136277/ef7e3598-242a-11e7-9c2f-a2675a72b79b.png)



## Trello card
https://trello.com/c/zUGiAMmY/1-maquetar-la-seccion-pricipal-de-la-pagina

SKOLZ commented on pagina.html:
```diff
@@ -0,0 +1,43 @@
+            All your debts are kept safe in your phone.
+          </p>
+        </div>
+        </div>
```
from this line to the end of the file the elements should have one tab less

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,122 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,700');
+*{
```
space before curly brace.

SKOLZ commented on styles.css:
```diff
@@ -0,0 +1,122 @@
+}
+
+.info {
+  font: Open Sans;
```
no need to add this here since you already added that property in the selector that's in line 2

gonziesc commented:
## Summary
- Deleted unnecesary font declaration
- Added space before curly brace
- Indentation fix con index.html

## Trello Card
https://trello.com/c/y13AH8Pk/24-maquetado

igna92ts commented on app/constants/index.js:
```diff
@@ -0,0 +1,3 @@
+const config = require('../../config');
+
+exports.apiBaseUrl = config.common.api.apiBaseUrl;
```
do you gain anything from doing this intead of just using it from the config?

igna92ts commented on app/controllers/album.js:
```diff
@@ -0,0 +1,18 @@
+    .then(response => res.status(response.statusCode).send(response.body))
+    .catch(err => {
+      logger.error(`Error getting albums: ${err}`);
+      return res.status(500).send(err);
```
shouldnt you send the error you get from the API instead of always returning 500? also use the errors from the `errors.js` file

gon-escalante replied:
if the request is made i return the message that it returns, whatever that is, if anything else fails i return 500

igna92ts commented on app/helpers/request.js:
```diff
@@ -0,0 +1,12 @@
+const request = require('request');
+
+module.exports = (options, handleError) =>
+  new Promise((resolve, reject) => {
```
I would rather have you use request-promise instead of this

igna92ts commented on app/services/album.js:
```diff
@@ -0,0 +1,13 @@
+  headers: { 'content-type': 'application/json' }
+});
+
+exports.getAlbums = () => request(buildDefaultApiConfig('albums'), error => new Error(error));
```
declare errors inside the `errors` file rather than using `new Error` every time

igna92ts commented on config/index.js:
```diff
@@ -36,7 +36,8 @@ const config = {
       parameterLimit: process.env.API_PARAMETER_LIMIT,
-      port: process.env.PORT
+      port: process.env.PORT,
+      apiBaseUrl: process.env.API_URL
```
this refers to the API you are making and if im not wrong `apiBaseUrl` is for an external api so I dont think both should be inside the `api` property in the config object

igna92ts commented on test/setup.js:
```diff
@@ -0,0 +1,29 @@
+
+beforeEach(done => {
+  const models = require('../app/models');
+
```
delete the empty lines here

igna92ts commented on test/setup.js:
```diff
@@ -0,0 +1,29 @@
+  truncateDatabase().then(() => done());
+});
+
+// including all test files
```
remove commented code
